chapters/part3/semantic-parsing-grammar.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
Probably better to keep these unfinished ones in the branch.

Reply:
Done.